apisix/plugins/basic-auth.lua

```diff
@@ -125,7 +125,7 @@ do
     end
 end
 
-function _M.access(conf, ctx)
+function _M.rewrite(conf, ctx)
```

Comment:
why do we need to change another plugin?

one PR for one thing, we should only update something about plugin `authz-keycloak`

please revert this code